### PR TITLE
[1LP][RFR] Adding test coverage for bugzilla

### PR DESCRIPTION
**Pull request by mnadeem92**
Signed-off-by: mnadeem92 <mnadeem@redhat.com>

### CODE REVIEW

File: cfme/tests/v2v/test_v2v_manual.py [hunk 1]
```diff
@@ -194,3 +194,113 @@ def test_osp_migration_request_details_page_shows_vms_for_not_started_plans():
...
+
+
+@pytest.mark.tier(2)
+def test_detection_non_uci_vm():
+    """
+    title: Test the non uci vm should not detect while configuring conversion host
+
+    Bugzilla:
+        1810091
+    Polarion:
```
Comment (ganeshhubale) [inline cfme/tests/v2v/test_v2v_manual.py:206]: Add BZ as with **coverage** marker.
Update other tests too.

File: cfme/tests/v2v/test_v2v_manual.py [hunk 2]
```diff
@@ -194,3 +194,113 @@ def test_osp_migration_request_details_page_shows_vms_for_not_started_plans():
...
+        assignee: mnadeem
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.11
+        testSteps:
```
Comment (ganeshhubale) [inline cfme/tests/v2v/test_v2v_manual.py:211]: **ignore** this test case for 5.10 version.
Update other tests too.

File: cfme/tests/v2v/test_v2v_manual.py [hunk 3]
```diff
@@ -194,3 +194,113 @@ def test_osp_migration_request_details_page_shows_vms_for_not_started_plans():
...
+            1. Create a CloudForms appliance with region 0, say cf1.example.com
+            2. Configuration the conversion host in cf1.example.com
+            3. Run a migration on cf1.example.com, the pod will be called conversion-1
+            4. Create another appliance with region 0, say cf.example.com
+            5. Configuration the conversion host in cf2.example.com
+            6. Run a migration on cf2.example.com
+
+        expectedResults:
```
Comment (ganeshhubale) [inline cfme/tests/v2v/test_v2v_manual.py:303]: remove blank line

File: cfme/tests/v2v/test_v2v_manual.py [hunk 4]
```diff
@@ -194,3 +194,113 @@ def test_osp_migration_request_details_page_shows_vms_for_not_started_plans():
...
+    Polarion:
+        assignee: mnadeem
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.11
+        testSteps:
+            1. Click mapping creation widget
+            2. Select non-admin project
+        expectedResults:
+            1. Mapping should create successfully
```
Comment (valaparthvi) [inline cfme/tests/v2v/test_v2v_manual.py:235]: No 2nd expectedResults? Maybe add a blank `2.` here.

Comment (valaparthvi) [reply cfme/tests/v2v/test_v2v_manual.py:235]: Same for other tests.

File: cfme/tests/v2v/test_v2v_manual.py (hunk 3 above)
Comment (valaparthvi) [reply cfme/tests/v2v/test_v2v_manual.py:316]: Which blank lines?

File: cfme/tests/v2v/test_v2v_manual.py (hunk 1 above)
Comment (valaparthvi) [reply cfme/tests/v2v/test_v2v_manual.py:209]: You haven't made this change @mnadeem92.

File: cfme/tests/v2v/test_v2v_manual.py [hunk 5]
```diff
@@ -2,6 +2,7 @@
 import pytest
 
 from cfme import test_requirements
+from cfme.utils.blockers import BZ
```
Comment (ganeshhubale) [inline cfme/tests/v2v/test_v2v_manual.py:6]: We don't need this import.

File: cfme/tests/v2v/test_v2v_manual.py (hunk 1 above)
Comment (ganeshhubale) [reply cfme/tests/v2v/test_v2v_manual.py:209]: @mnadeem92 
It should be like this - ```@pytest.mark.meta(coverage=[1810091])```

**Pull request merged**